geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/AbstractIngest.scala

```diff
@@ -118,7 +118,6 @@ abstract class AbstractIngest(val dsParams: Map[String, String],
           try {
             val (sft, features) = converter.convert(is)
             if (features.hasNext) {
-              ds.createSchema(sft)
```

Comment:
Can this actually go?  Seems like it'd be needed to create the schema?

Reply:
at some point we starting calling createSchema in the 'preIngest' method, so this isn't needed anymore